media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -276,66 +273,68 @@ <h3>
 zabezpečenie oprávnených záujmov dotknutej osoby).
 </p>
 
-<p>
+<p id="dalsie-prava-dotknutej-osoby-9-4">
 <span>9.4.</span>
 Dotknutá osoba má právo podať návrh na začatie konania o ochrane osobných údajov na Úrad na ochranu
 osobných údajov Slovenskej republiky pri podozrení, že jej osobné údaje sa neoprávnene spracúvajú. 
 </p>
 
-<p>
+<p id="dalsie-prava-dotknutej-osoby-9-5">
 <span>9.5.</span>
 Ak dotknutá osoba nemá spôsobilosť na právne úkony v plnom rozsahu, jej práva môže uplatniť zákonný
 zástupca.
 </p>
 
-<p>
+<p id="dalsie-prava-dotknutej-osoby-9-6">
 <span>9.6.</span>
 Ak dotknutá osoba nežije, je práva, ktoré mala podľa
 <a href="https://www.slov-lex.sk/pravne-predpisy/SK/ZZ/2013/122/?ucinnost=01.12.2015" target="_blank">Zákona</a>,
 môže uplatniť blízka osoba.
 </p>
 
-<p>
+<p id="dalsie-prava-dotknutej-osoby-9-7">
 <span>9.7.</span>
 Ak dotknutá osoba: 
 </p>
 
-<p>
+<p id="dalsie-prava-dotknutej-osoby-9-7-1">
 <span>9.7.1.</span>
-uplatní námietku podľa 9.2.1., je Prevádzkovateľ povinný spracúvané osobné údaje bez zbytočného
+uplatní námietku podľa <a href="#dalsie-prava-dotknutej-osoby-9-2-1">9.2.1.</a>, je Prevádzkovateľ
+povinný spracúvané osobné údaje bez zbytočného
```

Comment:
whitespace upravim az v poslednom PR v ramci 337, este sa to bude tak ci tak menit.